app/models/demographic_question.rb

```diff
@@ -9,6 +9,7 @@
 
 class DemographicQuestion < ApplicationRecord
   has_many :answers, as: :answerable
+  has_many :options
```

Comment:
Don't forget we want to make this optional

Reply:
Ah yes, thanks @spencerldixon I will fix that.

Reply:
I read the Rails docs and it appears that the `optional` is an option for `belongs_to` so I will put it inside the `Options` model. I checked `has_many` and it does not have an `optional`.